core/version.go

```diff
@@ -1,7 +1,7 @@
 package core
 
 // VERSION number of current image server
-const VERSION = "1.18.1"
+const VERSION = "1.18.2"
```

Comment:
how about making this `1.19.0` as you are adding new functionality?

Reply:
```suggestion
const VERSION = "1.19.0"
```